docs/content/preview/api/ycql/dml_transaction.md

```diff
@@ -32,13 +32,14 @@ transaction_block ::= BEGIN TRANSACTION
                       END TRANSACTION ';'
 ```
 
-Where
+Where `insert`, `update`, and `delete` are [INSERT](../dml_insert), [UPDATE](../dml_update/), and [DELETE](../dml_delete/) statements.
 
-- `insert`, `update`, and `delete` are [INSERT](../dml_insert), [UPDATE](../dml_update/), and [DELETE](../dml_delete/) statements.
+- When using `BEGIN TRANSACTION`, you don't use a semicolon. End the transaction block with `END TRANSACTION ;`.
+- There is no `COMMIT` for transactions started using `BEGIN`.
 
-### ANSI SQL syntax
+### SQL syntax
 
-Alternatively, YugabyteDB supports ANSI SQL `START TRANSACTION` and `COMMIT` statements.
+YCQL also supports SQL `START TRANSACTION` and `COMMIT` statements.
 
 ```
```

Comment:
For glorious syntax highlighting, ebnf is the language for all these. :)

````suggestion
```ebnf
````